polaris-react/src/components/Checkbox/Checkbox.scss

```diff
@@ -40,14 +40,11 @@
     + .Backdrop {
       // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
       @include focus-ring($style: 'focused');
-
-      #{$se23} & {
-        outline: var(--p-border-width-2) solid
-          var(--p-color-border-interactive-focus);
-        outline-offset: var(--p-space-025);
-        background-color: var(--p-color-bg-input-hover-experimental);
-        border-width: 0;
-      }
+      outline: var(--p-border-width-2) solid
+        var(--p-color-border-interactive-focus);
+      outline-offset: var(--p-space-025);
+      background-color: var(--p-color-bg-input-hover-experimental);
+      border-width: 0;
```

Comment:
* [This PR](https://5d559397bae39100201eedc1-tehbqainqm.chromatic.com/?path=/story/all-components-checkbox--default)
* [Production](https://storybook.polaris.shopify.com/?path=/story/all-components-checkbox--default&globals=polarisSummerEditions2023:true)

<img width="926" alt="Screenshot 2023-08-25 at 1 48 56 PM" src="https://github.com/Shopify/polaris/assets/20652326/16bd2fe8-2477-49a8-94d1-2a57de433690">